libblkid/src/superblocks/drbd.c

```diff
@@ -132,6 +138,9 @@ static int probe_drbd_84(blkid_probe pr, const struct blkid_idmag *mag)
 	if (!md)
 		return errno ? -errno : 1;
 
+	if (md->bm_bytes_per_bit != BM_BLOCK_SIZE)
```

Comment:
Should use both `read_unaligned_member()` and `be32_to_cpu()`, similar to `device_uuid` below.

Please make sure the test `blkid/low-probe` succeeds under ASAN.
(CI is currently broken unfortunately, should be fixed soon)

Reply:
Thank you for review, updated!